src/components/forms/CreateForm.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+import List from "../formLists/List";
+import { TextField, Stack, Button } from "@mui/material";
+
+const title = { fontSize: "13px", fontWeight: "bold" };
+
+const CreateForm = ({ setActivityDetails, setIsTransactionFormShown }) => {
+  const [activity, setActivity] = useState("");
+  const [contributors, setcontributors] = useState("");
+  const [items, setItems] = useState([]); //items refer to an array of contributors
+
+  const addcontributors = (event) => {
+    event.preventDefault();
+    if (!contributors) {
+      return;
+    }
+    setItems([...items, contributors]);
```

Comment:
I don't really get why you have 2 states storing the same data 🤔 I am pretty sure we could manage this in one state variable somehow.